doc/api/next_api_changes/removals/26965-ER.rst

```diff
@@ -0,0 +1,20 @@
+``matplotlib.cm.register_cmap``
```

Comment:
You probably will want to copy what's in https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.7.0.html#deprecation-of-top-level-cmap-registration-and-access-functions-in-mpl-cm